pages/contribute/rule-feedback.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Give feedback on rules
+---
+
+Feedback on rules is encouraged on our open [GitHub repository](https://github.com/act-rules/act-rules.github.io), so you don't even have to be a member of the Community Group to contribute in this way.
+
+If you "watch" the repository, you will get notifications when changes are happening.
+
+## Feedback for ideas and early drafts (Issues on GitHub)
+
+Rules and Definitions (shared terms across rules) that are GitHub Issues can be everything from just a title to a quite finished rule. The issues will be prefixed with either "Rule:" (or possibly "New Rule:") or "Algorithm".
```

Comment:
Are algorithms the same as definitions? Even for someone fairly familiar with things this paragraph isn't very clear.

Reply:
```suggestion
Draft rules and definitions (shared terms across rules) that are currently GitHub issues can be anything from just a title to an almost finished rule. These issues will be prefixed with either "Rule:" (or possibly "New Rule:") or "Algorithm".
```

Reply:
I'm also not sure what "Algorithm" refers to in our process 🤔

Reply:
Yikes! Yeah "algorithm" is what we used to call definitions, when rules were would prescribe the exact procedure to follow.